### ISSUE TITLE
QUESTION: no documentation?

### ISSUE THREAD

**Issue by joonjeong** [faust-streaming/faust#271]
Hi, I get to know faust.

sadly, faust looks no updates anymore but I check faust-streaming is working on

Is there no document like [faust](https://faust.readthedocs.io/)?

**Comment by AntonOfTheWoods** [comment]
@joonjeong , I think the reason is that there haven't really been any improvements or new functionality added, just making sure dependencies can be updated and security/stability fixes.

**Comment by joonjeong** [comment]
@AntonOfTheWoods Thank you for your comment!

I checked there's no big changes on faust-streaming but some features are not updated on upstream documents.

For example, it looks that prometheus sensor is committed to master but it's not applied, and github version tag has 1.10.3 and it's also published on pip but upstream document looks still based on 1.9.x.

I know upstream project exists, but can't check it's status and roadmap, so I got a little confused which package I have to use. So I just understand that robinhood still has ownership of project(not certain...) and this is why there's no documentation of this fork...
